jsonnet/kube-prometheus/kube-prometheus-thanos.libsonnet

```diff
@@ -43,7 +43,7 @@ local servicePort = k.core.v1.service.mixin.spec.portsType;
           '--log.level=debug',
           '--query.replica-label=prometheus_replica',
           '--query.auto-downsampling',
-          '--store=dnssrv+thanos-sidecar.' + $._config.namespace + '.svc',
+          '--store=dnssrv+thanos-store.' + $._config.namespace + '.svc:10901',
```

Comment:
we need both sidecar and store here no? cc @metalmatze

Reply:
The reference is to the DNS based SD for finding the store service. The sidecar service was acting as the proxy to Prometheus in the last update. It should instead point to the Store service.

It was also missing the GRPC port of the store service.

Reply:
I mean we want to discover both sidecars as well as stores, so both flags, instead of removing the sidecar one:

```
          '--store=dnssrv+thanos-sidecar.' + $._config.namespace + '.svc',
          '--store=dnssrv+thanos-store.' + $._config.namespace + '.svc:10901',
```

Reply:
Yes, we want both. Additionally, we also want to use the proper way of using dnssrv records.
Maybe you could update that in this PR too?

https://github.com/metalmatze/kube-thanos/blob/master/manifests/thanos-querier-deployment.yaml#L22